### DISCUSSION TITLE
QT配置环境问题

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#169] [Q&A] [answered]
使用的是QT6.2.4，遇到如下问题
![IM(RC CB 1PJ3~`2K8@%6YP](https://user-images.githubusercontent.com/100075314/163667859-e61c318f-689d-4fa2-ac16-b6a34a63e157.png)
按照网上搜到的在.pro文件中增加QT+=...完全没有解决问题，特此求助各位

**Comment by liusy58** [comment]
仅凭借一个截图很难定位你的错误 ;-(

**Reply by VansamaZWT** [reply]
您需要别的什么来帮助判断错误吗，因为代码没有实质内容所以只截了第一行报错
![image](https://user-images.githubusercontent.com/100075314/163676944-ea41db91-ebba-480d-b573-055c8aa16128.png)

**Reply by liusy58** [reply]
Q戳我把 ;-)

**Reply by liusy58** [reply]
看起来比较像是版本问题，可以试一下5.14.2？

**Comment by VansamaZWT** [comment, marked as answer]
问题已解决，感谢助教liusy58的帮助，请不要在路径中使用中文=.=
